[0m18:47:56.444851 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x73f0e2ad3950>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x73f0e29b6840>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x73f0e2c46930>]}


============================== 18:47:56.462522 | b170f118-c26f-456c-af4a-33ac37efe1e0 ==============================
[0m18:47:56.462522 [info ] [MainThread]: Running with dbt=1.9.0
[0m18:47:56.463121 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': '/home/cpkv/.dbt', 'log_path': 'logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt init', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m18:47:56.471731 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object, got 'PosixPath'.. Dictionary: {'dir': PosixPath('/home/cpkv/.dbt')}
[0m18:47:56.472188 [info ] [MainThread]: Creating dbt configuration folder at 
[0m18:48:21.660934 [debug] [MainThread]: Starter project path: /home/cpkv/cpkv/venv/lib/python3.12/site-packages/dbt/include/starter_project
[0m18:48:21.673612 [info ] [MainThread]: 
Your new dbt project "custom_postgres" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m18:48:21.675662 [info ] [MainThread]: Setting up your profile.
[0m19:09:29.520085 [info ] [MainThread]: Profile custom_postgres written to /home/cpkv/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m19:09:29.531346 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 1293.183, "process_in_blocks": "2992", "process_kernel_time": 0.108232, "process_mem_max_rss": "93432", "process_out_blocks": "88", "process_user_time": 1.379964}
[0m19:09:29.535448 [debug] [MainThread]: Command `dbt init` succeeded at 19:09:29.534650 after 1293.19 seconds
[0m19:09:29.538302 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x73f0e25de5a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x73f0e34dd370>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x73f0e34dce00>]}
[0m19:09:29.541090 [debug] [MainThread]: Flushing usage events
[0m19:09:31.053018 [debug] [MainThread]: An error was encountered while trying to flush usage events
